docs/TabbedFormEnterprise.md

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
You should not introduce a new chapter, but rather add an RBAC section to the existing chapter. Example: https://marmelab.com/react-admin/SimpleForm.html#role-based-access-control-rbac